components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -560,7 +560,7 @@ SUBROUTINE p3_main_part2(kts, kte, kbot, ktop, kdir, do_predict_nc, dt, inv_dt,
       if (qi(k).ge.qsmall) log_exitlevel = .false.
       !enddo
       if (log_exitlevel .and.                                                           &
-         (t(k).lt.zerodegc .and. qv_supersat_i(k).lt.-0.05_rtype)) goto 555   !i.e. skip all process rates
+         (t_atm(k).lt.T_zerodegc .and. qv_supersat_i(k).lt.-0.05_rtype)) goto 555   !i.e. skip all process rates
```

Comment:
In these comparisons we have lowercase "t"_atm compared with uppercase "T"_zerodegc.  Does it make it more readable to stick with a single case "T" to represent anything related to temperature?

Reply:
Yeah, we should use a consistent case. It would probably be best to change t_atm to T_atm everywhere. We might as well change th_atm to Th_atm as well for consistency.